WeHere/Views&ViewModels/Components/Modal/BottomModal/BottomModal.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  BottomModal.swift
+//  WeHere
+//
+//  Created by Chanhee Jeong on 2022/11/21.
+//
+
+import SwiftUI
+
+struct BottomModal: View {
+    @State private var currentIndex = 0
+    let place = Place.mockData[1]
+    
+    var body: some View {
+        TabView(selection: $currentIndex) {
+            ForEach(0..<2) { index in
+                ZStack {
+                    Color.white
+                    switch index {
+                    case 0:
+                        pokeView
+                            .padding(EdgeInsets(top: 28, leading: 28, bottom: 28, trailing: 28))
```

Comment:
```suggestion
                            .padding(28)
```
[P1] 4방위가 28로 동일하다면 위와 같이 줘도 좋을 것 같습니다.

Reply:
바꾸는걸 까먹었네요! 감사합니다
8d5a909ff75a88ebedba1ff0a99237f79aebb42b